.github/workflows/pr.yml

```
@@ -0,0 +1,140 @@
+
+            Make sure to choose \`luxmeter-esp8266.factory.bin\`.
+
+            [download]: ${url}`
+            })
+
+  comment-vindriktning:
+    if: github.event_name == 'pull_request'
+    name: Comment on PR
+    runs-on: ubuntu-latest
+    needs:
+        - build-vindriktning
```

yamllint

[warning] wrong indentation: expected 6 but found 8

(indentation)